monkey/common/configuration/agent_sub_configurations.py

```diff
@@ -12,6 +12,26 @@ class CustomPBAConfiguration:
 
 @dataclass(frozen=True)
 class PluginConfiguration:
+    """
+    A configuration for plugins
+
+    :param name: Name of the plugin.
```

Comment:
Don't put periods at the end of these. See examples in https://guardicore.atlassian.net/wiki/spaces/RES/pages/2800353435/Infection+Monkey+Development+Guidelines+DRAFT#Docstrings-Format